nex/nex.go

```diff
@@ -122,6 +122,9 @@ func init() {
 	rootfs.Flag("image", "Base image for rootfs build").Default("synadia/nex-rootfs:alpine").StringVar(&RootfsOpts.BaseImage)
 	rootfs.Flag("agent", "Path to agent binary").PlaceHolder("../path/to/nex-agent").Required().StringVar(&RootfsOpts.AgentBinaryPath)
 	rootfs.Flag("size", "Size of rootfs filesystem").Default(strconv.Itoa(1024 * 1024 * 150)).IntVar(&RootfsOpts.RootFSSize) // 150MB default
+
+	// one day when we refactor, let's get rid of all of these global structs. Such ugly
+	nodesLs.Flag("workload", "Only query nodes currently running the given workload (id or name)").StringVar(&RunOpts.Name)
```

Comment:
Maybe add a placeholder with a workloadID?

Reply:
if I do that it'll just look like a nuid. and you can use either a nuid or the workload name... Given that it felt like I shouldn't have a placeholder